Woof/WoofTests/ViewModel/DetailPetSitterViewModelTests.swift

```diff
@@ -0,0 +1,37 @@
+import XCTest
+
+final class DetailPetSitterViewModelTests: XCTestCase {
+    func testDetailPetSitterViewModelExistsInAPI() {
+        let sitter = Sitter(
+            id: UUID(),
+            name: "Ann",
+            surname: "Anderson",
+            phone: "132451",
+            bio: "Some random text",
+            rating: 4.7,
+            pricePerHour: 14
+        )
+
+        _ = DetailSitterViewModel(sitter: sitter)
+    }
+
+    func testDetailPetSitterViewModelPropertiesReturnsExpectedValues() {
+        let sitter = Sitter(
+            id: UUID(),
+            name: "Ann",
+            surname: "Anderson",
+            phone: "132451",
+            bio: "Some random text",
+            rating: 4.0,
+            pricePerHour: 14
+        )
+
+        let viewModel = DetailSitterViewModel(sitter: sitter)
+        
+        XCTAssertEqual(viewModel.fullName, "\(sitter.name) \(sitter.surname)")
+        XCTAssertEqual(viewModel.phoneNumber, sitter.phone)
+        XCTAssertEqual(viewModel.bio, sitter.bio)
+        XCTAssertEqual(viewModel.rating, 4)
```

Comment:
nit:

Shouldn't that be:
```suggestion
        XCTAssertEqual(viewModel.rating, sitter.rating)
```
?

Reply:
We expected here 4, because we transformed data and it should be integer instead of double.

Reply:
Then you can make it type-compatible:
```swift
XCTAssertEqual(viewModel.rating, Int(sitter.rating))
```
или так:
```swift
XCTAssertEqual(viewModel.rating, DataTrasnformer.normalizeRating(sitter.rating))
````

Reply:
Thanks, fixed it.